firmware/integration/rusefi_config.txt

```diff
@@ -1348,6 +1348,7 @@ custom stepper_num_micro_steps_e 1 bits, U08,	@OFFSET@,	[0:3], @@stepper_num_mic
     brain_input_pin_e[SENT_INPUT_COUNT iterate] sentInputPins;
 
     int8_t launchFuelAdderPercent
+	uint8_t autoscale etbJamTimeout;Time required to detect a stuck throttle.;"sec", 0.02, 0, 0, 5, 2
 	int8_t[1 iterate] sorryUnused
```

Comment:
i have a feeling @mck1117 was meaning to remove "sorryUnused" field

while waiting for CHANGELOG I've managed to use same unused config area so now we have https://github.com/rusefi/rusefi/pull/4894